src/common/util/submit-data.js

```diff
@@ -57,7 +57,13 @@ submitData.xhr = function xhr (url, body, sync, method = 'POST') {
     // do nothing
   }
 
-  request.setRequestHeader('content-type', 'text/plain')
+  if (gzipped) {
+    request.setRequestHeader('content-type', 'application/json')
+    request.setRequestHeader('X-Browser-Monitoring-Key', licenseKey)
```

Comment:
Is this still necessary? Adding this header causes CORS options calls to be made to our edge network and is not compatible with `sendBeacon`.